coordinator/src/test/scala/filodb.coordinator/queryplanner/LogicalPlanParserSpec.scala

```diff
@@ -57,6 +57,7 @@ class LogicalPlanParserSpec extends AnyFunSpec with Matchers {
     parseAndAssertResult("""count_values("freq",http_requests_total{job="app"})""")
     parseAndAssertResult("""timestamp(http_requests_total{job="app"})""")
     parseAndAssertResult("""absent(http_requests_total{job="app"})""")
+    parseAndAssertResult("""absent(sum(http_requests_total{job="app"}))""")
```

Comment:
Can you add similar test cases for other types of functions that are supported with absent like sum_over_time, exp?